views/backend/list.php

```diff
@@ -1,30 +1,24 @@
-<?php
-    if($model_user->admin()!=1){
-        header("Location:/dashboard");
-    }
-?>
 <h2>Listing des articles</h2>
 <hr/>
 
 <?php
-$posts = $model_article->get_posts();
     foreach($posts as $post){
         ?>
         <div class="row">
             <div class="col s12">
-                <h4><?= $post->title ?><?php echo ($post->posted == "0") ? "<i class='material-icons'>lock</i>" : "" ?></h4>
+                <h4><?= $post->getTitle() ?><?php echo ($post->getPosted() == "0") ? "<i class='material-icons'>lock</i>" : "" ?></h4>
                 <div class="row">
                     <div class="col s12 m6 l8">
-                        <?= substr(nl2br($post->chapo),0,1200) ?>...
+                        <?= substr(nl2br($post->getChapo()),0,1200) ?>...
                     </div>
                     <div class="row"></div>
                     <div class="col s12 m6 l8">
-                        <?= substr(nl2br($post->content),0,1200) ?>...
+                        <?= substr(nl2br($post->getContent()),0,1200) ?>...
                     </div>
                     <div class="col s12 m6 l4">
-                        <img src="public/img/posts/<?= $post->image ?>" class="materialboxed responsive-img" alt="<?= $post->title ?>"/>
+                        <img src="public/img/posts/<?= $post->getImage() ?>" class="materialboxed responsive-img" alt="<?= $post->getTitle() ?>"/>
                         <br/><br/>
-                        <a class="btn light-blue waves-effect waves-light" href="modifier-un-article/<?= transforme_en_url($post->title) ?>-<?= $post->id ?>">Modifier l'article</a>
+                        <a class="btn light-blue waves-effect waves-light" href="modifier-un-article/<?= transforme_en_url($post->getTitle()) ?>-<?= $post->getId() ?>">Modifier l'article</a>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$post'.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)